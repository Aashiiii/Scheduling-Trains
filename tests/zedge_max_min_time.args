Werrington Westmead 23:59:59
