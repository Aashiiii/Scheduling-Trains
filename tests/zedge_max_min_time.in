Campsie::Vineyard::23:59:59
Campsie::Waitara::07:12:19
Canterbury::Windsor::03:12:25                                                                                                                                                   
Clyde::Windsor::11:12:11
Werrington::Westmead::00:00:00
Waverton::Windsor::20:00:01
Werrington::Vineyard::03:45:12
