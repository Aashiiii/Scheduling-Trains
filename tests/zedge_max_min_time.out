The next train to Westmead from Werrington departs at 00:00:00
